cvmfs/swissknife_ingest.cc

```diff
@@ -152,7 +155,8 @@ int swissknife::Ingest::Main(const swissknife::ArgumentList &args) {
   }
 
   sync->Traverse();
-
+  printf("ingest......................................\n");
```

Comment:
Even if the PR is WIP, let's not keep `printf` statements around. Please use the `LogCvmfs` function.